source/utilities/config.ts

```diff
@@ -0,0 +1,126 @@
+// source/utilities/config.ts
+// Parse and return the configuration for the CLI.
+
+import {
+  resolve as resolvePath,
+  relative as resolveRelativePath,
+} from 'node:path';
+import { readFile } from 'node:fs/promises';
+import Ajv from 'ajv';
+// @ts-expect-error No type definitions.
+import schema from '@zeit/schemas/deployment/config-static.js';
+import { resolve } from './promise.js';
+import type { ErrorObject } from 'ajv';
+import type { Configuration, Options, NodeError } from '../types.js';
+
+/**
+ * Parses and returns a configuration object from the designated locations.
+ *
+ * @param cwd - The current working directory.
+ * @param entry - The directory to serve.
+ * @param args - The arguments passed to the CLI.
+ *
+ * @returns The parsed configuration.
+ */
+export const loadConfiguration = async (
+  cwd: string,
+  entry: string,
+  args: Options,
+): Promise<Configuration> => {
+  const files = ['serve.json', 'now.json', 'package.json'];
```

Comment:
There was an error thrown in the previous file is you use `now.json` since it's deprecated - I wonder if we rip that logic out?

Reply:
e.g. if we wrote a test, I don't think we'd ever hit this logic block.

Reply:
It will only log a warning to the console if you use `package.json` or `now.json`, not throw an error.

I could remove support for these two locations, but we would then need to make a semver major release as it would be a breaking change.

Reply:
Yeah, agreed. Let's just leave them 👍